tests/test_send_stats_to_performance_platform.py

```diff
@@ -0,0 +1,359 @@
+import json
```

Comment:
Hmm, yep maybe I should have thought of that - you mean just set up the data responses in a mock client, hit `send_framework_stats` and then check the right calls got made to performance platform?  That would have been nice.  I can change it if you reckon it's worth it now, but I'm not sure it's worth re-doing.